MAINTAINERS.yml

```diff
@@ -111,6 +111,19 @@
 
 # Areas are sorted by name
 
+ACPI:
+  status: maintained
+  maintainers:
+    - jhedberg
```

Comment:
There's a separate section for the acipca west module too, for which I'm a de-facto maintainer (I think I'm also the only one with push access) - could you add me there as well? Thanks.

Reply:
this is different from the west project.

Reply:
> this is different from the west project.

I know. I was asking if you could update the section for the west project in the same go, since right now I'm not listed there.

Reply:
Anyway, not absolutely mandatory to be done in this PR, so I'll approve and let you decide.